desktop_version/src/Game.h

```diff
@@ -390,6 +391,8 @@ class Game
     void returntoeditor();
     bool shouldreturntoeditor;
 #endif
+
+    int gametimer = 0;
```

Comment:
Initializing things in headers is C++11 only. This should be `int gametimer;`. (And there should be a `gametimer = 0;` in `Game::init()`, but technically you don't need to do that because you've already initialized it in `main()` anyway.)